documentation/src/main/asciidoc/mapstruct-reference-guide.asciidoc

```diff
@@ -2109,6 +2111,16 @@ However, by specifying `nullValueMappingStrategy = NullValueMappingStrategy.RETU
 The strategy works in a hierarchical fashion. Setting `nullValueMappingStrategy` on mapping method level will override `@Mapper#nullValueMappingStrategy`, and `@Mapper#nullValueMappingStrategy` will override `@MappingConfig#nullValueMappingStrategy`.
 
 
+[[mapping-result-for-null-properties]]
```

Comment:
I don't think that this is only for `null`, it should also be valid for presence check methods